src/pages/Lend/admin/FeaturedAdminControls.vue

```diff
@@ -1,192 +0,0 @@
-<template>
```

Comment:
I don't know what this file is, but it is not referenced anywhere and hasn't been touched since 2018. It is the only place that'@/graphql/query/featuredLoansControl.graphql' and '@/graphql/mutation/setFeaturedLoans.graphql' are referenced, so I deleted those files too

Reply:
This was all part of an embedded experiment/setting managment tool. I'd left the files in place b/c it was a really nice interface for the purpose, but we can revisit later if we need to set something like that up again.